.github/workflows/flutter.yml

```diff
@@ -42,6 +46,14 @@ jobs:
             distribution: 'adopt'
             java-version: '8'
 
+        # Install required dependencies for Flutter on Linux on Ubuntu
+        - name: 'Setup Linux'
+          run: |
+            sudo apt update
+            sudo apt install -y cmake dbus libblkid-dev libgtk-3-dev liblzma-dev ninja-build pkg-config xvfb
+            sudo apt install -y network-manager upower
```

Comment:
is this really necessary? I don't see as a requirement here https://flutter.dev/docs/get-started/install/linux#install-flutter-manually

Reply:
The first few builds failed because at least ninja was not available on the ubuntu image. I then looked up what the Flutter Plus Plugin maintainers did. I haven't yet tried to reduce the number of installed packages.

I think we can drop at least `sudo apt install -y network-manager upower` but I have yet to try it.

Reply:
it was just curiosity tbh, if the flutter plus plugin does the same, let's keep like this :)
I've not installed any of those on my ubuntu machine and it worked out ootb, but I might have them installed already

Reply:
I think there's also a difference between Ubuntu Desktop and Ubuntu Server